.github/workflows/build-wheels.yml

```
@@ -0,0 +1,31 @@
+name: Build wheels
+
+on:
+  workflow_dispatch
+
+jobs:
+  buildwheels:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        python-version: [3.6]
```

Nitpick: I recommend making this a string, having recently fallen into the trap of adding a Python version of `3.10` rather than `'3.10'` in the Traits workflows.
```suggestion
        python-version: ['3.6']
```